BraveWallet/Panels/Add Suggested Network/AddSuggestedNetworkView.swift

```diff
@@ -0,0 +1,136 @@
+// Copyright 2022 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import BraveUI
+import struct Shared.Strings
+import BraveShared
+import BraveCore
+
+struct AddSuggestedNetworkView: View {
+  @ObservedObject var keyringStore: KeyringStore
+  @ObservedObject var networkStore: NetworkStore
+  var chain: BraveWallet.NetworkInfo
+  
+  @ScaledMetric private var blockieSize = 24
+  @ScaledMetric private var faviconSize = 48
+  
+  @Environment(\.openWalletURLAction) private var openWalletURL
+  
+  private var headerView: some View {
+    VStack {
+      HStack(spacing: 8) {
+        Spacer()
+        Text(keyringStore.selectedAccount.address.truncatedAddress)
```

Comment:
No, the origin in this PR is hard-coded and not setup, but it will be added here: https://github.com/brave/brave-ios/pull/5164/files#diff-2e77a33043f377f9a58af58bcbb41bbc53d06985612b101c42062a1abe146bc0R90

Reply:
How many lines will it contain? Does it wrap and scroll if too many are contained?

Reply:
As many as it needs to display the entire address, all UI is contained inside scrollable containers by default